k8s/aat/common/divorce/div-fps.yaml

```diff
@@ -28,9 +28,7 @@ spec:
       useInterpodAntiAffinity: true
       applicationInsightsInstrumentKey: "63c30ef9-a27b-4d38-8afc-4c180fb6ac33"
       image: hmctspublic.azurecr.io/div/fps:prod-5e4711bb
-      ingressHost: div-fps-aat.service.core-compute-aat.internal
       environment:
-        FEE_API_URL: "http://fees-register-api-aat.service.core-compute-aat.internal"
```

Comment:
https://github.com/hmcts/div-fees-and-payments-service/blob/master/charts/div-fps/values.yaml#L5